manifests/base/controller/deployment.yaml

```diff
@@ -41,10 +41,16 @@ spec:
         ports:
         - containerPort: 3000
           protocol: TCP
+        volumeMounts:
+        - name: patch-file
+          mountPath: ~/
```

Comment:
This needs to be absolute path other than HOME (this would overwrite the app runtime files), use something like `/mnt/config`